superset/models/helpers.py

```diff
@@ -33,6 +35,24 @@ def json_to_dict(json_str):
     else:
         return {}
 
+def has_kerberos_ticket():
+    IS_KERBEROS_ENABLED = get_env_variable('IS_KERBEROS_ENABLED')
+
+    if IS_KERBEROS_ENABLED is not None and subprocess.call(['klist', '-s']) == 0:
+        subprocess.call(['/usr/local/bin/auth-kerberized.sh'])
+
+def get_env_variable(var_name, default=None):
```

Comment:
why we need this?

Reply:
used to get enviroment variable which is set at deployment time to check if superset is enabled with kerberos access or not.